packages/bpk-component-breakpoint/src/BpkBreakpoint.js

```diff
@@ -29,11 +31,18 @@ const BREAKPOINTS = {
   ABOVE_TABLET: TOKENS.breakpointQueryAboveTablet,
 };
 
-const BpkBreakpoint = props => (
-  <MediaQuery query={props.query}>{props.children}</MediaQuery>
-);
+type Props = {
+  children: Node | (() => mixed),
```

Comment:
This file doesn't import `type Node` from React, so this is probably referring to the wrong Node.

Reply:
Hmm it's strange that it somehow still accepted it though!

Reply:
Yeah, seemed weird but I figured I should fix it anyway!

Also: ➡️ 🌴 ❗